caravel/tasks.py

```diff
@@ -0,0 +1,43 @@
+import celery
+from caravel import db, models, app
+from sqlalchemy import select, text
+from sqlalchemy.sql.expression import TextAsFrom
+import pandas as pd
+
+celery_app = celery.Celery(config_source=app.config.get('CELERY_CONFIG'))
+
+def get_sql_results_sync(database_id, sql):
+    session = db.session()
+    mydb = session.query(models.Database).filter_by(id=database_id).first()
+
+    content = ""
+    if mydb:
+        eng = mydb.get_sqla_engine()
+        if app.config.get('SQL_MAX_ROW'):
+            sql = sql.strip().strip(';')
+            qry = (
+                select('*')
+                .select_from(TextAsFrom(text(sql), ['*']).alias('inner_qry'))
+                .limit(app.config.get('SQL_MAX_ROW'))
+            )
+            sql = str(qry.compile(eng, compile_kwargs={"literal_binds": True}))
+        try:
+            df = pd.read_sql_query(sql=sql, con=eng)
+            content = df.to_html(
```

Comment:
This was kind of a hack to build the feature quickly, eventually we should just go `to_json` here.